integration-test/README.md

```diff
@@ -0,0 +1,49 @@
+## Integration Tests
+
+This test folder creates a tarball of the main repository and runs the main tests in a browser context with multiple bundlers and tools. The goal is to ensure that someone downstream of the package will not have an issue when importing it. 
+
+Note: currently we assume that there is one entry point for the main tests. 
+
+## Bundlers and Tools Tested
+
+Currently, we test:
+* webpack
+* browserify
+* rollup
+* parcel 
+* mocked version of unpkg (this tests that the `dist/nat.umd.js` file works in the browser as if it came from unpkg. We can't use the actual unpkg because it would test the version currently published to npm, not our local version.)
+
+For more information on how widely each tool is used, see the [2018 State of JS survey](https://2018.stateofjs.com/other-tools/#build_tools). 
+
+## Methods
+
+Note: The actual commands can be found in the `test_integration` job in the CircleCI config (`.circlci/config.yml`).
+
+We start with the unit tests that we already have in `test/test.js` of the main directory. Then, we transform the test file into a few files that will be used by the bundlers:
+* a ES6 modules version that replaces the Nat local import in the test file (`import Nat from '../src/index';`) with an import of the package (`@agoric/nat`)
+* a CommonJS version that does the same replacement (this will be used by browserify, which cannot process ES6 modules)
+* a CommonJS version that removes all Nat imports (this will be used by unpkg)
```

Comment:
maybe add a bit, "removes all Nat imports in favor of a global 'Nat' object that would be put in place by a previous '<script>' tag"